File: PosseUpApp/app/src/androidTest/java/com/lordan/mark/PosseUp/MainActivityTest.java
package com.lordan.mark.PosseUp;

import android.os.SystemClock;

import android.support.test.espresso.ViewAssertion;
import android.support.test.espresso.assertion.ViewAssertions;
import android.support.test.rule.ActivityTestRule;
import android.support.test.runner.AndroidJUnit4;
import android.view.View;

import com.lordan.mark.PosseUp.UI.MainActivityGroup.MainActivity;

import org.junit.Rule;
import org.junit.Test;
import org.junit.runner.RunWith;

import java.util.concurrent.TimeUnit;

import static android.support.test.espresso.Espresso.onView;
import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
import static android.support.test.espresso.matcher.ViewMatchers.withText;

/**
 * Created by Mark on 26/04/2016.
 */
@RunWith(AndroidJUnit4.class)
public class MainActivityTest {
    @Rule
    public final ActivityTestRule<MainActivity> main = new ActivityTestRule<>(MainActivity.class);

    @Test
    public void launchMainActivity() {
        onView(withText("Classic car show")).check(ViewAssertions.matches(isDisplayed()));
    }
}

File: PosseUpApp/app/src/androidTest/java/com/lordan/mark/PosseUp/MainActivityTest.kt
package com.lordan.mark.PosseUp

import android.os.SystemClock

import android.support.test.espresso.ViewAssertion
import android.support.test.espresso.assertion.ViewAssertions
import android.support.test.rule.ActivityTestRule
import android.support.test.runner.AndroidJUnit4
import android.view.View

import com.lordan.mark.PosseUp.UI.MainActivityGroup.MainActivity

import org.junit.Rule
import org.junit.Test
import org.junit.runner.RunWith

import java.util.concurrent.TimeUnit

import android.support.test.espresso.Espresso.onView
import android.support.test.espresso.matcher.ViewMatchers.isDisplayed
import android.support.test.espresso.matcher.ViewMatchers.withText

/**
 * Created by Mark on 26/04/2016.
 */
@RunWith(AndroidJUnit4::class)
class MainActivityTest {
    @Rule @JvmField val main = ActivityTestRule<MainActivity>(MainActivity::class.java)

    @Test
    fun launchMainActivity() {
        onView(withText("Classic car show")).check(ViewAssertions.matches(isDisplayed()))
    }
}
